### DISCUSSION TITLE
issues when deploying your own website please

### DISCUSSION THREAD

**Discussion by XiCheng129** [academicpages/academicpages.github.io#2457] [Q&A]
when I was attempting to push a change to the template repository, issues occur with "pages build and deployment / build (dynamic) Failing after 19s". Thanks a lot.

**Comment by rjzupkoii** [comment]
When this happens you need to go into the logs for the build to see what happens - in this case the build you are referring to [is here](https://github.com/XiCheng129/XiCheng129.github.io/actions/runs/10965398009/job/30451113546). Looking at the output the following stands out:

```
 Error: could not read file /github/workspace/_publications/2009-10-01-paper-title-number-1.md: Invalid date '2025': Document '_publications/2009-10-01-paper-title-number-1.md' does not have a valid date in the YAML front matter.
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    Invalid date '2025': Document '_publications/2009-10-01-paper-title-number-1.md' does not have a valid date in the YAML front matter.
```

Based upon the file in question it looks like you have just the year for the date, but the parser is expected the year, month, and day. Adding the month and day to the date should fix the error.
